### PR TITLE
Creating file for Ezoic integrated sites

### PR DESCRIPTION
**Pull request by jbauer23**
[Ezoic](https://www.ezoic.com) is a Google Certified Publishing Partner. Our Partner Manager is Mauricio Geutierrez (molvera@google.com), and he told us about this program. 

Our service for publishers has improves ad management, site speed, and a few other aspects of their site. Because one of our features improves site speed by fixing issues with these audits, we'd like to submit a stack pack so that publishers on our system can be notified to turn on relevant features. I've submitted recommendations for 13 audits and we have plans to support at least 2 more in the near future. 

To detect if a site is using Ezoic, look for the `X-Middleton-Display` response header. Its presence indicates that a site is running on our platform.

### CODE REVIEW

**Comment by housseindjirdeh** [comment]
Sorry for taking so long to get to this! We've been prioritizing other CMS/frameworks that have more usage, but we can definitely look to include this :)

Just two things:

1. Would you be able to add the detection to the [library detector](https://github.com/johnmichel/Library-Detector-for-Chrome/blob/master/library/libraries.js)? (separate repo)
2. Can you update this to follow the [newer format](https://github.com/GoogleChrome/lighthouse-stack-packs/blob/master/packs/drupal.js) we have (`.js` files with an export)

**Comment by housseindjirdeh** [comment]
Friendly ping @jbauer23! If you can't make any of the changes, no worries and let me know :)

**Comment by jbauer23** [comment]
Sorry, been crazy and this got lost in the shuffle. I'll take a look this week.

**Comment by housseindjirdeh** [comment]
No rush at all, the apologies are all on my end for taking so long to get to this

**Comment by jbauer23** [comment]
Updating the detector will take a little work on our end, so I'm passing this off to someone else on the team to finish up.

**Comment by connorjclark** [comment]
Hi @jbauer23, if you are still interested: FYI the library detector runs inside a page, thus it cannot rely on headers. Is there some JS global that this CMS exposes? If not, could you add one?

**Comment by jbauer23** [comment]
Hey @connorjclark I've got our pack updated to the current format here, but it needs approval from a maintainer. 

We also have another PR that has the updates to the detector library here: https://github.com/johnmichel/Library-Detector-for-Chrome/pull/193

File: packs/ezoic.js
Comment (connorjclark) [inline packs/ezoic.js:6]: you can delete all the meta-comments above this line

File: packs/ezoic.js
Comment (connorjclark) [inline packs/ezoic.js:31]: for this string and all others, change up the grammar to be more like:

`Use Ezoic Leap and enable `Some-Feature` to convert images to WebP`

basically, avoid present participles (adding -ing to verbs) and prefer the normal form of verb, as it tends to simplify the language

File: packs/ezoic.js
Comment (connorjclark) [inline packs/ezoic.js:39]: just delete these empty ones

**Review by connorjclark — CHANGES_REQUESTED** [review]

File: packs/ezoic.js
Comment (jbauer23) [reply packs/ezoic.js]: I've made those changes, thanks.

File: packs/ezoic.js
Comment (connorjclark) [inline packs/ezoic.js:25]:
```suggestion
  'uses-rel-preconnect': 'Use [Ezoic Leap](https://pubdash.ezoic.com/speed) and enable `Pre-Connect Origins` to automatically add `preconnect` resource hints to establish early connections to important third-party origins.',
```

File: packs/ezoic.js
Comment (connorjclark) [inline packs/ezoic.js:29]:
```suggestion
  'font-display': 'Use [Ezoic Leap](https://pubdash.ezoic.com/speed) and enable `Optimize Fonts` to automatically leverage the `font-display` CSS feature to ensure text is user-visible while webfonts are loading.',
```

**Pull request merged**
